airbyte-cdk/python/airbyte_cdk/sources/declarative/incremental/datetime_based_cursor.py

```diff
@@ -52,7 +52,7 @@ class DatetimeBasedCursor(Cursor):
     datetime_format: str
     config: Config
     parameters: InitVar[Mapping[str, Any]]
-    _cursor: str = field(repr=False, default=None)  # tracks current datetime
+    _cursor: Optional[str] = field(repr=False, default=None)  # tracks current datetime
```

Comment:
Was this type wrong, or are we explicitly changing it to be `Optional`?

Reply:
It was typed wrong